src/verifier.js

```diff
@@ -145,5 +146,11 @@ module.exports = function (options) {
 		checkTypes.assert.string(options.providerBaseUrl);
 	}
 
-	return new Verifier(options.providerBaseUrl, options.pactUrls, options.providerStatesUrl, options.providerStatesSetupUrl, options.pactBrokerUsername, options.pactBrokerPassword);
+	if (options.timeout) {
+		checkTypes.assert.positive(options.timeout);
+	} else {
+		options.timeout = 30000;
```

Comment:
Could you move this up to the start of the function and have the default there like `options.timeout = options.timeout || 30000`.  Leave the conditional/assert there though.